server/server.go

```diff
@@ -38,14 +42,13 @@ func CreateServer() *Server {
 }
 
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	s.Router.ServeHTTP(w, r)
-}
-
-func (s *Server) Serve() error {
-	return http.ListenAndServe("localhost:18080", s.Router)
+	logger.HttpLogger(s.Router).ServeHTTP(w, r)
 }
 
 func (s *Server) Routes() {
 	s.Router.HandleFunc("/user", s.HandleRegisterUser()).Methods("POST")
 	s.Router.HandleFunc("/user/login", s.HandleLogin()).Methods("POST")
+	s.Router.HandleFunc("/user/active", s.HandleGetAvtiveUsers()).Methods("GET")
+	wrappedMux := logger.HttpLogger(s.Router)
+	log.Fatal(http.ListenAndServe("localhost:18080", wrappedMux))
```

Comment:
Just a reminder: Please avoid of using the hardcoded values. host name and port should be defined in a variables and (ideally) should be configurable via system variables or command line arguments